apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx

```diff
@@ -118,8 +112,8 @@ const RegisterPage = () => {
                 {intl.formatMessage(
                   {
                     defaultMessage:
-                      "GCKey is a service that allows you to log into <a>many different government services</a> with a single username and password. The button below will take you to GCKey’s website, where you’ll be able to create an account if you don’t already have one. Once you’ve registered for GCKey, you’ll be brought back here to complete your profile.",
-                    id: "cPgH94",
+                      "GCKey is a service that allows you to sign in to <a>many different government services</a> with a single username and password. The button below will take you to GCKey's website, where you'll be able to create an account if you don't already have one. Once you've signed up for GCKey, you'll be brought back here to complete your profile.",
```

Comment:
"button below" is using directional language.

Reply:
Out of scope, [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/7375) created.